#
# Alga message
#

# Position of the corners of the alga
geometry_msgs/Point32 p1
geometry_msgs/Point32 p2
geometry_msgs/Point32 p3
geometry_msgs/Point32 p4

# Heatmap
Float32Array[] disease_heatmap
